addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -31,7 +31,7 @@
                         <div t-attf-class="mb-3 row o_url_input#{state.isButton ? ' d-none' : ''}">
                             <label class="col-form-label col-md-3" for="o_link_dialog_url_input">URL or Email</label>
                             <div class="col-md-9">
-                                <input type="text" name="url" class="form-control" id="o_link_dialog_url_input" required="required" t-ref="inputUrl"/>
+                                <input type="text" name="url" class="form-control" id="o_link_dialog_url_input" required="required" t-ref="inputUrl" t-on-keydown="(ev) => ev.key === 'Enter' ? this.onSave(ev) : ''"/>
```

Comment:
I think in owl you can name the property `t-on-keydown.prevent` to imply a `preventDefault` so you won't have to add it in the js above. Try it and let me know if it worked.

Reply:
I want to prevent the `keydown` event for the Enter key only. However, using `t-on-keydown.prevent` prevents all `keydown` events, making it impossible to type anything in the URL input box.

Reply:
Oh, right, thanks for trying!